imgui/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ if (MSVC)
     target_sources(imgui PRIVATE ${imgui_srcdir}/examples/imgui_impl_directx11.cpp)
 else()
     if (APPLE)
+        target_compile_options(imgui PRIVATE -fobjc-arc)
         target_sources(imgui PRIVATE ${imgui_srcdir}/examples/imgui_impl_metal.mm)
```

Comment:
Let's do the same target properties here as for wpigui.  Also, while you're here, let's add `target_compile_features(imgui PUBLIC cxx_std_17)`